packages/@n8n/task-runner/src/config/sentry-config.ts

```diff
@@ -0,0 +1,21 @@
+import { Config, Env } from '@n8n/config';
+
+@Config
+export class SentryConfig {
+	/** Sentry DSN */
+	@Env('N8N_SENTRY_DSN')
+	sentryDsn: string = '';
+
+	//#region Metadata about the environment
+
+	@Env('N8N_VERSION')
+	n8nVersion: string = '';
+
+	@Env('ENVIRONMENT')
+	environment: string = '';
```

Comment:
Some env vars have `N8N_` and others don't?

Reply:
I wanted to keep them 1:1 with what we use in the n8n app